huawen-lei/week4/wed/homework-w4d3/views/results.erb

```diff
@@ -0,0 +1,47 @@
+
+<h2>Movie Info of "<%= @movie_list %>"</h2>
+
+<hr>
+
+Movie Title: <%= @movie_title %>
+<br>
+Release Date: <%= @movie_year %>
+<br>
+<img src="<%= @movie_img %>" alt="movie_poster">
+
+
+<hr>
+
+<h3>List version:</h3>
+
+<ul>
+<% @movie_array.each do |item| %>
+
+    <li>
+
+    <%# tltle %>
+    <a href="/movie/<%=item["id"] %>">
+    <h4><%= item["title"] %></h4>
+    </a>
```

Comment:
why is this clickable? it takes me to a page where it is just the ID number. Did you want to take me to a different place?